### PR TITLE
Use readKeyFile to read both seed file and public key file

### PR DESCRIPTION
**Pull request by nanjj**
It's consistent with the key reading behavior in `sign`, and we can add some comments in both seed file and public key file.

### CODE REVIEW

**Review by derekcollison — COMMENTED** [review]
Update copyright to 2018-2023 then I think it LGTM

**Comment by nanjj** [comment]
> Update copyright to 2018-2023 then I think it LGTM

Fixed

File: nk/main.go [hunk 1]
```diff
@@ -157,7 +157,7 @@ func sign(fname, keyFile string) {
 	if err != nil {
 		log.Fatal(err)
 	}
-	log.Printf("%s", base64.StdEncoding.EncodeToString(sigraw))
+	log.Printf("%s", base64.RawURLEncoding.EncodeToString(sigraw))
```
Comment (derekcollison) [inline nk/main.go:160]: Why the change here?

Comment (nanjj) [reply nk/main.go:160]: Because in JWT three parts, header.payload.signature, the signature part is encoded in base64 url no padding format according JWT spec. If `nk` uses base64 url no padding format as signature content,  `nk` command can be used to verify jwt token like this: 
    nk -verify header.payload -pubin pubin -sigfile signature
Or nk would compain illegle charset error.

**Review by derekcollison — APPROVED** [review]
LGTM
